virttest/utils_test/libvirt.py

```diff
@@ -3619,6 +3619,28 @@ def check_qemu_cmd_line(content, err_ignore=False):
     return True
 
 
+def check_cmd_output(cmd, content, err_ignore=False):
+    """
+    Check the specified content in the output of the cmd
+    :param cmd: the cmd wants to check
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
+    :raise: exceptions, TestFail when content not found in the output of cmd
```

Comment:
Minor typo fix.
exceptions, TestFail => exceptions.TestFail when content is not found